jsk_pcl_ros_utils/src/polygon_magnifier_nodelet.cpp

```diff
@@ -72,8 +72,18 @@ namespace jsk_pcl_ros_utils
   {
     boost::mutex::scoped_lock lock(mutex_);
     vital_checker_->poke();
+
     jsk_recognition_msgs::PolygonArray ret_polygon_array;
     ret_polygon_array.header = msg->header;
+
+    ret_polygon_array.labels.reserve(msg->labels.size());
+    std::copy(msg->labels.begin(), msg->labels.end(),
+              std::back_inserter(ret_polygon_array.labels));
+
+    ret_polygon_array.likelihood.reserve(msg->likelihood.size());
+    std::copy(msg->likelihood.begin(), msg->likelihood.end(),
+              std::back_inserter(ret_polygon_array.likelihood));
```

Comment:
@wkentaro Thanks for reviewing! What is done in this part is just to copy all elements of vector.